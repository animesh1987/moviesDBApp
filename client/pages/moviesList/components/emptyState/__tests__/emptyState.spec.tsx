import * as React from 'react';
import { create } from 'react-test-renderer';

import { EmptyState } from '../index';

describe('Empty State Test', () => {

  it('renders correctly', () => {
    const component = create(
      <EmptyState />)
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
