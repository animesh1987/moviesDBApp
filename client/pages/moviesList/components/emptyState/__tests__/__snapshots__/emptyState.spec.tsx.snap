// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Empty State Test renders correctly 1`] = `
<div
  className="empty-state"
>
  <h1>
    No films found
  </h1>
</div>
`;
